Core/ConfigWizard.cs

```diff
@@ -0,0 +1,208 @@
+using System;
+using System.IO;
+using System.Text;
+using LocalAdmin.V2.IO;
+
+namespace LocalAdmin.V2.Core
+{
+    public static class ConfigWizard
+    {
+        public static void RunConfigWizard()
+        {
+            var curConfig = LocalAdmin.Configuration;
+            
+            Retry:
+            Console.WriteLine("Welcome to LocalAdmin Configuration Wizard!");
+            Console.WriteLine();
+            Console.WriteLine(
+                "We will ask you a couple of questions. You can always change your answers by running LocalAdmin with \"--reconfigure\" argument or manually editing configuration files in " +
+                LocalAdmin.GameUserDataRoot + "config directory.");
+            Console.WriteLine();
+            
+            Console.WriteLine(LocalAdmin.Configuration == null ? "This is the default LocalAdmin configuration:" : "That's your current LocalAdmin configuration:");
+            LocalAdmin.Configuration ??= new Config();
+            Console.WriteLine(LocalAdmin.Configuration.ToString());
+            
+            Console.WriteLine();
+            var input = string.Empty;
+            while (input == null || !input.Equals("edit", StringComparison.OrdinalIgnoreCase) &&
+                   !input.Equals("keep", StringComparison.OrdinalIgnoreCase))
+            {
+                Console.Write("Do you want to edit that configuration? [edit/keep]: ");
```

Comment:
Would be nice to have keep as default option, so when you press enter it just auto selects keep.